[2025-01-28T02:25:41.928+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T02:25:42.006+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.task1 scheduled__2025-01-16T00:00:00+00:00 [queued]>
[2025-01-28T02:25:42.072+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.task1 scheduled__2025-01-16T00:00:00+00:00 [queued]>
[2025-01-28T02:25:42.075+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-01-28T02:25:42.230+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): task1> on 2025-01-16 00:00:00+00:00
[2025-01-28T02:25:42.252+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1229) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-28T02:25:42.264+0000] {standard_task_runner.py:72} INFO - Started process 1243 to run task
[2025-01-28T02:25:42.245+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'task1', 'scheduled__2025-01-16T00:00:00+00:00', '--job-id', '85', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmps5twempt']
[2025-01-28T02:25:42.276+0000] {standard_task_runner.py:105} INFO - Job 85: Subtask task1
[2025-01-28T02:25:42.527+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag.task1 scheduled__2025-01-16T00:00:00+00:00 [running]> on host 6d5b1ca1fccb
[2025-01-28T02:25:43.004+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2025-01-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-16T00:00:00+00:00'
[2025-01-28T02:25:43.007+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T02:25:43.084+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-28T02:25:43.087+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello World']
[2025-01-28T02:25:43.107+0000] {subprocess.py:99} INFO - Output:
[2025-01-28T02:25:43.110+0000] {subprocess.py:106} INFO - Hello World
[2025-01-28T02:25:43.112+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-28T02:25:43.173+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T02:25:43.177+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=task1, run_id=scheduled__2025-01-16T00:00:00+00:00, execution_date=20250116T000000, start_date=20250128T022542, end_date=20250128T022543
[2025-01-28T02:25:43.230+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T02:25:43.270+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-28T02:25:43.274+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
